prow/config.yaml

```diff
@@ -610,6 +637,52 @@ presubmits:
         emptyDir: {}
 
   kubernetes/cloud-provider-azure:
+  - name: pull-cloud-provider-azure-e2e
+    agent: kubernetes
+    always_run: false
+    context: pull-cloud-provider-azure-e2e
+    rerun_command: "/test pull-cloud-provider-azure-e2e"
+    trigger: "(?m)^/test pull-cloud-provider-azure-e2e,?(\\s+|$)"
+    branches:
+    - master
+    labels:
+      preset-service-account: true
+      preset-cloud-provider-azure-account: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180511-51cf02e75-master
+        args:
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--timeout=100"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        env:
+        - name: DOCKER_IN_DOCKER_ENABLED
+          value: 'true'
+        securityContext:
+          privileged: true
+          capabilities:
+            add: ["SYS_ADMIN"]
```

Comment:
I don't think you need SYS_ADMIN here...? The other dind jobs have done fine with just `privileged`.